src/components/uikit/CallToAction/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { styled } from 'styled-components';
+import { IoHelpBuoyOutline } from 'react-icons/io5';
+
+const CallToAction = () => (
+  <CTAContainer>
+    <Layer>
+      <ContentContainer>
+        <CTATitle>We are all close together, be toxic</CTATitle>
+        <CTADescription>
+          A problem, a question, an emergency?
+          <br />
+          Do not hesitate to visit the help centre, we can help you
+        </CTADescription>
+        <HelpCenterButton>
+          <IoHelpBuoyOutline />
+          Help center
+        </HelpCenterButton>
+      </ContentContainer>
+    </Layer>
+  </CTAContainer>
+);
+
+const Layer = styled.div`
```

Comment:
Greate Work Mohammed 🔥, I think its good to separate style in external file such as **style.ts**

Reply:
@Batrawi 
Mornings 💛
I told you before to create the style in the same file but I think it's better if separate this to two files🙏

Reply:
ok ok, thanks for your comments 🤝
@karamzomlot  @KhaledAref13